### ISSUE TITLE
Errors starting docker with molecule

### ISSUE THREAD

**Issue by bibliotechy** [tulibraries/ansible-role-docker-server-bootstrap#1]
This role errors out on testing, which seems to be related to running docker in the molecule docker driver.

Errors from `journalctl -u docker.service`
```
May 20 17:26:12 instance dockerd[6563]: time="2019-05-20T17:26:12.596300264Z" level=error msg="'overlay2' is not supported over aufs" sto
rage-driver=overlay2
May 20 17:26:12 instance dockerd[6563]: time="2019-05-20T17:26:12.597057610Z" level=error msg="AUFS is not supported over aufs" storage-d
river=aufs
May 20 17:26:12 instance dockerd[6563]: time="2019-05-20T17:26:12.597728051Z" level=error msg="'overlay' is not supported over aufs" stor
age-driver=overlay
May 20 17:26:12 instance dockerd[6563]: time="2019-05-20T17:26:12.751031002Z" level=warning msg="Usage of loopback devices is strongly di
scouraged for production use. Please use `--storage-opt dm.thinpooldev` or use `man dockerd` to refer to dm.thinpooldev section." storage
-driver=devicemapper
May 20 17:26:12 instance dockerd[6563]: time="2019-05-20T17:26:12.853291706Z" level=warning msg="XFS is not supported in your system (exe
c: \"mkfs.xfs\": executable file not found in $PATH). Defaulting to ext4 filesystem" storage-driver=devicemapper
May 20 17:26:12 instance dockerd[6563]: time="2019-05-20T17:26:12.861877797Z" level=info msg="Creating filesystem ext4 on device docker-0
:57-8430-base, mkfs args: [/dev/mapper/docker-0:57-8430-base]" storage-driver=devicemapper
May 20 17:26:12 instance dockerd[6563]: time="2019-05-20T17:26:12.862050325Z" level=info msg="Error while creating filesystem ext4 on dev
ice docker-0:57-8430-base: exec: \"mkfs.ext4\": executable file not found in $PATH" storage-driver=devicemapper
May 20 17:26:12 instance dockerd[6563]: time="2019-05-20T17:26:12.862227905Z" level=warning msg="Failed to deactivatePool: Device is Busy
" storage-driver=devicemapper
May 20 17:26:12 instance dockerd[6563]: Error starting daemon: error initializing graphdriver: exec: "mkfs.ext4": executable file not fou
nd in $PATH
May 20 17:26:12 instance systemd[1]: docker.service: main process exited, code=exited, status=1/FAILURE
May 20 17:26:12 instance systemd[1]: Failed to start Docker Application Container Engine.
May 20 17:26:12 instance systemd[1]: Unit docker.service entered failed state.
```

**Comment by cmharlow** [comment]
@bibliotechy do we leave this to recall the issue; or just consider #2 to have closed it? It somewhat makes sense to me that we'd use a Vagrant test setup for running docker install testing, since we wouldn't try to run docker within docker in reality / production.

**Comment by bibliotechy** [comment]
I'm calling this closed by #2
